compiler/rustc_feature/src/active.rs

```diff
@@ -426,6 +426,8 @@ declare_features! (
     (active, intra_doc_pointers, "1.51.0", Some(80896), None),
     /// Allows `#[instruction_set(_)]` attribute
     (active, isa_attribute, "1.48.0", Some(74727), None),
+    // Uses keyword generics for ~const bounds
+    (active, keyword_generics, "1.62.0", Some(102090), None),
```

Comment:
yea. I want to get all tests passing with the new feature and then retry libcore and keep adding tests for libcore failures until everything works. This means we can merge the PR once an initial set of tests passes instead of having this PR open forever.